### DISCUSSION TITLE
Having issues with swift installation from all image references after April 10th.

### DISCUSSION THREAD

**Discussion by LokeshNagireddy** [actions/virtual-environments#9802] [General questions] [answered]
Hi Team,

im having issues with swift installation as shown in below error.

==> azure-arm.build_image: Provisioning with shell script: /azp/_work/1/s/runner-images/images/ubuntu/templates/../scripts/build/install-swift.sh
==> azure-arm.build_image: Downloading package from https://swift.org/builds/swift-5.10-release/ubuntu2204/swift-5.10-RELEASE/swift-5.10-RELEASE-ubuntu22.04.tar.gz to /tmp/swift-5.10-RELEASE-ubuntu22.04.tar.gz...
==> azure-arm.build_image: Package downloaded in 13 seconds
==> azure-arm.build_image: gpg: directory '/root/.gnupg' created
==> azure-arm.build_image: gpg: keybox '/root/.gnupg/pubring.kbx' created
==> azure-arm.build_image: **gpg: keyserver receive failed: Connection timed out**

im not having proper support from the runner-images team.

Can someone please help me to solve this issue. The above issue is not happening from the image commit of April 10th. But it have a different issue with install-packer.

Successful log of Swift from image commit of 10th April.
Downloading package from https://swift.org/builds/swift-5.10-release/ubuntu2204/swift-5.10-RELEASE/swift-5.10-RELEASE-ubuntu22.04.tar.gz to /tmp/swift-5.10-RELEASE-ubuntu22.04.tar.gz...
Package downloaded in 8 seconds
Downloading package from https://swift.org/keys/all-keys.asc to /tmp/all-keys.asc...
Package downloaded in 1 seconds
gpg: directory '/root/.gnupg' created
gpg: keybox '/root/.gnupg/swift' created
gpg: /root/.gnupg/trustdb.gpg: trustdb created
gpg: key D441C977412B37AD: public key "Swift Automatic Signing Key #1 <swift-infrastructure@swift.org>" imported
gpg: key 9F597F4D21A56D5F: public key "Swift 2.2 Release Signing Key <swift-infrastructure@swift.org>" imported
gpg: key 63BC1CFE91D306C6: public key "Swift 3.x Release Signing Key <swift-infrastructure@swift.org>" imported
gpg: key EF5430F071E1B235: public key "Swift 4.x Release Signing Key <swift-infrastructure@swift.org>" imported
gpg: key 7638F1FB2B2B08C4: public key "Swift Automatic Signing Key #2 <swift-infrastructure@swift.org>" imported
gpg: key 925CC1CCED3D1561: public key "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" imported
gpg: key FAF6989E1BC16FEA: public key "Swift Automatic Signing Key #3 <swift-infrastructure@swift.org>" imported
gpg: key 925CC1CCED3D1561: "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" 1 new signature
gpg: key F167DF1ACF9CE069: public key "Swift Automatic Signing Key #4 <swift-infrastructure@forums.swift.org>" imported
gpg: key 925CC1CCED3D1561: "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" 1 new signature
gpg: key F167DF1ACF9CE069: "Swift Automatic Signing Key #4 <swift-infrastructure@forums.swift.org>" 1 new signature
gpg: Total number processed: 11
gpg:               imported: 8
gpg:         new signatures: 3
Downloading package from https://swift.org/builds/swift-5.10-release/ubuntu2204/swift-5.10-RELEASE/swift-5.10-RELEASE-ubuntu22.04.tar.gz.sig to /tmp/swift-5.10-RELEASE-ubuntu22.04.tar.gz.sig...
Package downloaded in 1 seconds
gpg: Signature made Tue Mar  5 19:06:05 2024 UTC
gpg:                using RSA key 925CC1CCED3D1561
gpg: Good signature from "Swift 5.x Release Signing Key <swift-infrastructure@swift.org>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: A62A E125 BBBF BB96 A6E0  42EC 925C C1CC ED3D 1561
SWIFT_PATH=/usr/share/swift/usr/bin
Pester v5.5.0

Starting discovery in 1 files.
Discovery found 10 tests in 205ms.
Filter 'FullName' set to ('Swift').
Filters selected 3 tests to run.

**Comment by LokeshNagireddy** [comment]
@erik-bershel , This issue is still happening with any of the latest updates from main branch. 
Can you please help me to solve this issue

**Comment by LokeshNagireddy** [comment]
@erik-bershel, May you please clarify this issue. why swift installation working with some commit, and not working with latest commit. Surely you changed something after the commit https://github.com/actions/runner-images/commit/3afd153301d1a498c8d2be9c7abfed7cfa8e5498.

I look forward for your response.

**Comment by erik-bershel** [comment, marked as answer]
Related issue with an answer: https://github.com/actions/runner-images/issues/9788
